runtime/oti/j2sever.h

```diff
@@ -67,13 +67,13 @@
 #elif JAVA_SPEC_VERSION == 9
 	#define J2SE_SHAPE_LATEST       J2SE_SHAPE_B165
 #else
-	#define J2SE_SHAPE_LATEST       J2SE_SHAPE_B1803
+	#define J2SE_SHAPE_LATEST       J2SE_SHAPE_B1000
 #endif
 #define J2SE_SHAPE_SUN     		0x10000
 #define J2SE_SHAPE_B136    		0x40000
 #define J2SE_SHAPE_B148    		0x50000
 #define J2SE_SHAPE_B165    		0x60000
-#define J2SE_SHAPE_B1803		0x70000
+#define J2SE_SHAPE_B1000		0x70000
```

Comment:
We could change the name to something like J2SE_SHAPE_V10. In the future if build levels are also required then we would add J2SE_SHAPE_V10_B24

Reply:
We could also rename the other shapes like J2SE_SHAPE_B136 -> J2SE_SHAPE_V9_B136 and J2SE_SHAPE_B165 -> J2SE_SHAPE_V9

Reply:
> I feel these Java 9 renaming could be done in a separate cleanup PR instead.
Agreed

Reply:
J2SE_SHAPE_B1000 should be renamed as part of this PR, are you doing that?

Reply:
Updated now, pls have another look.